packages/node_modules/@webex/internal-plugin-calendar/test/unit/spec/utils.js

```diff
@@ -0,0 +1,13 @@
+import {assert} from '@webex/test-helper-chai';
+
+import CalendarUtils from '../../../src/util';
+
+describe('Calendar utils', () => {
+  describe('#calculateEndTime()', () => {
+    it('return the end time for meeting', () => {
+      const result = CalendarUtils.calculateEndTime({start: '2020-02-21T18:57:00.000Z', durationMinutes: 30});
+
+      assert.equal(new Date(result.endTime).toISOString(), new Date('Fri Feb 21 2020 14:27:00 GMT-0500 (Eastern Standard Time)').toISOString());
```

Comment:
Could you use the same date format so the time zone won't come into play?

```suggestion
      assert.equal(new Date(result.endTime).toISOString(), new Date('2020-02-21T19:27:00.000Z').toISOString());
```